openslides/utils/autoupdate.py

```diff
@@ -411,18 +64,21 @@ def inform_deleted_data(elements: Iterable[Tuple[str, int]], information: Dict[s
 
     The argument information is added to each collection element.
     """
-    # Put all stuff to be deleted into the autoupdate_bundle.
```

Comment:
Here the same as in inform_changed_data().

Reply:
done